zulip_bots/zulip_bots/bots/joinme/joinme.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any
+import requests
+import json
+import socket
+from six.moves.configparser import SafeConfigParser
+
+class JoinmeHandler(object):
+    def usage(self) -> str:
+        return '''
+         A bot that will post a link to joinme call. Mention the bot and it will give out instructions.
+        '''
+
+    def handle_message(self, message: Any, bot_handler: Any) -> None:
+        self.config_info = bot_handler.get_config_info('joinme')
+        if 'ZISSHUDBUNIq' not in message['content']:  # The state, a special string to identify which type of url it is.
+            payload = {'client_id': self.config_info['key'], 'scope': 'user_info scheduler start_meeting',
+                       'redirect_uri': self.config_info['callback_url'],
+                       'state': 'ZISSHUDBUNIq', 'response_type': 'code'}
+            r = requests.get('https://secure.join.me/api/public/v1/auth/oauth2', params=payload)
+            if str(r) == '<Response [200]>':
```

Comment:
Use `r.status_code` here.

Reply:
But that is not going to work for the bot test, as the response in the test is in form of json, when the actual website response isn't.